test/qbwc/integration/session_test.rb

```diff
@@ -126,5 +126,16 @@ def requests(job, session, data)
     assert_equal 100, susan_session.progress
   end
 
+  test "session ticket generated does not collide with others generated in the same second" do
+    QBWC.add_job(:session_test_1, true, COMPANY, ConditionalTestWorker)
 
+    100.times do
```

Comment:
If we had access to `rspec-mocks` (or some other mocking framework) we could possibly mock `Time.now` so that we wouldn't need to iterate like this.